.github/scripts/tests/kind-integration.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -e
+
+if [ "$GIT_WORKSPACE" = "" ]; then
+    echo "GIT_WORKSPACE variable not definied. Should be the root of the source code. Example GIT_WORKSPACE=/home/dev/git/data-science-pipelines-operator" && exit
+fi
+
+if [ "$REGISTRY_ADDRESS" = "" ]; then
```

Comment:
small typo, *defined.

Reply:
commit message containing type I mean typo
it will be squashed, so nothing to worry